# English translations for vigilo-vigiconf.
# Copyright (C) 2011 ORGANIZATION
# This file is distributed under the same license as the vigilo-vigiconf
# project.
# FIRST AUTHOR <EMAIL@ADDRESS>, 2011.
#
msgid ""
msgstr ""
"Project-Id-Version: vigilo-vigiconf 2.0.0\n"
"Report-Msgid-Bugs-To: contact@projet-vigilo.org\n"
"POT-Creation-Date: 2011-02-24 16:47+0100\n"
"PO-Revision-Date: 2011-02-24 16:47+0100\n"
"Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
"Language-Team: en <LL@li.org>\n"
"Plural-Forms: nplurals=2; plural=(n != 1)\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=utf-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Generated-By: Babel 0.9.4\n"

#: src/vigilo/vigiconf/commandline.py:74
#, python-format
msgid "ERROR: %s"
msgstr "ERROR: %s"

#: src/vigilo/vigiconf/commandline.py:77
msgid "No server to manage."
msgstr "No server to manage."

#: src/vigilo/vigiconf/commandline.py:143
msgid ""
"Vigilo server management is only available in the Enterprise edition. "
"Aborting."
msgstr ""
"Vigilo server management is only available in the Enterprise edition. "
"Aborting."

#: src/vigilo/vigiconf/commandline.py:167
msgid "usage: "
msgstr "usage: "

#: src/vigilo/vigiconf/commandline.py:168
#, python-format
msgid "conflicting option string(s): %s"
msgstr "conflicting option string(s): %s"

#: src/vigilo/vigiconf/commandline.py:169
msgid "subcommands"
msgstr "subcommands"

#: src/vigilo/vigiconf/commandline.py:170
#, python-format
msgid "unrecognized arguments: %s"
msgstr "unrecognized arguments: %s"

#: src/vigilo/vigiconf/commandline.py:171
msgid "expected one argument"
msgstr "expected one argument"

#: src/vigilo/vigiconf/commandline.py:172
msgid "expected at most one argument"
msgstr "expected at most one argument"

#: src/vigilo/vigiconf/commandline.py:173
msgid "expected at least one argument"
msgstr "expected at least one argument"

#: src/vigilo/vigiconf/commandline.py:174
#, python-format
msgid "expected %s argument(s)"
msgstr "expected %s argument(s)"

#: src/vigilo/vigiconf/commandline.py:175
#, python-format
msgid "invalid choice: %r (choose from %s)"
msgstr "invalid choice: %r (choose from %s)"

#: src/vigilo/vigiconf/commandline.py:176
msgid "positional arguments"
msgstr "positional arguments"

#: src/vigilo/vigiconf/commandline.py:177
msgid "optional arguments"
msgstr "optional arguments"

#: src/vigilo/vigiconf/commandline.py:178
msgid "too few arguments"
msgstr "too few arguments"

#: src/vigilo/vigiconf/commandline.py:179
#, python-format
msgid "argument %s is required"
msgstr "argument %s is required"

#: src/vigilo/vigiconf/commandline.py:180
#, python-format
msgid "one of the arguments %s is required"
msgstr "one of the arguments %s is required"

#: src/vigilo/vigiconf/commandline.py:181
#, python-format
msgid "%s: error: %s\n"
msgstr "%s: error: %s\n"

#: src/vigilo/vigiconf/commandline.py:182
msgid "show this help message and exit"
msgstr "show this help message and exit"

#: src/vigilo/vigiconf/commandline.py:183
msgid "show program's version number and exit"
msgstr "show program's version number and exit"

#: src/vigilo/vigiconf/commandline.py:187
msgid "Debug mode."
msgstr "Debug mode."

#: src/vigilo/vigiconf/commandline.py:189
msgid "Don't try to switch to the correct user."
msgstr "Don't try to switch to the correct user."

#: src/vigilo/vigiconf/commandline.py:194
msgid "Vigilo configuration manager"
msgstr "Vigilo configuration manager"

#: src/vigilo/vigiconf/commandline.py:195
msgid "Subcommands"
msgstr "Subcommands"

#: src/vigilo/vigiconf/commandline.py:201
msgid "Prints a summary of the current configuration."
msgstr "Prints a summary of the current configuration."

#: src/vigilo/vigiconf/commandline.py:204
msgid "Supervision servers to query, all of them if not specified."
msgstr "Supervision servers to query, all of them if not specified."

#: src/vigilo/vigiconf/commandline.py:248
msgid "Deploys the configuration on each server if the configuration has changed."
msgstr "Deploys the configuration on each server if the configuration has changed."

#: src/vigilo/vigiconf/commandline.py:252
msgid "Stop the process after the file generation, before pushing the files."
msgstr "Stop the process after the file generation, before pushing the files."

#: src/vigilo/vigiconf/commandline.py:255
msgid ""
"Stop after pushing the files to the remote servers, and before restarting"
" the services."
msgstr ""
"Stop after pushing the files to the remote servers, and before restarting"
" the services."

#: src/vigilo/vigiconf/commandline.py:258
msgid "Deploy the given revision"
msgstr "Deploy the given revision"

#: src/vigilo/vigiconf/commandline.py:260
msgid ""
"Force the immediate execution of the command. Do not wait. Bypass all "
"checks."
msgstr ""
"Force the immediate execution of the command. Do not wait. Bypass all "
"checks."

#: src/vigilo/vigiconf/commandline.py:263
msgid "Simulate only, no copy will actually be made, no commit in the database."
msgstr "Simulate only, no copy will actually be made, no commit in the database."

#: src/vigilo/vigiconf/commandline.py:266
msgid "Supervision servers to deploy to, all of them if not specified."
msgstr "Supervision servers to deploy to, all of them if not specified."

#: src/vigilo/vigiconf/commandline.py:273
msgid "Discover the services available on a remote server."
msgstr "Discover the services available on a remote server."

#: src/vigilo/vigiconf/commandline.py:278
msgid "Output file. Default: standard output."
msgstr "Output file. Default: standard output."

#: src/vigilo/vigiconf/commandline.py:280
#, python-format
msgid "SNMP community. Default: %(default)s."
msgstr "SNMP community. Default: %(default)s."

#: src/vigilo/vigiconf/commandline.py:282
#, python-format
msgid "SNMP version. Default: %(default)s."
msgstr "SNMP version. Default: %(default)s."

#: src/vigilo/vigiconf/commandline.py:284
#, python-format
msgid "Main group. Default: %(default)s."
msgstr "Main group. Default: %(default)s."

#: src/vigilo/vigiconf/commandline.py:286
msgid ""
"Hosts or files to scan. The files must be the result of an snmpwalk "
"command on the '.1' OID with the '-OnQe' options."
msgstr ""
"Hosts or files to scan. The files must be the result of an snmpwalk "
"command on the '.1' OID with the '-OnQe' options."

#: src/vigilo/vigiconf/commandline.py:294
msgid "Enables or disables a Vigilo server"
msgstr "Enables or disables a Vigilo server"

#: src/vigilo/vigiconf/commandline.py:297
msgid "Don't re-deploy the configuration, only set the status."
msgstr "Don't re-deploy the configuration, only set the status."

#: src/vigilo/vigiconf/commandline.py:300
msgid "New status"
msgstr "New status"

#: src/vigilo/vigiconf/commandline.py:302
msgid "Server name(s) to enable/disable"
msgstr "Server name(s) to enable/disable"

#: src/vigilo/vigiconf/commandline.py:313
msgid ""
"VigiConf was launched as user 'root'. Switching to user 'vigiconf' "
"instead."
msgstr ""
"VigiConf was launched as user 'root'. Switching to user 'vigiconf' "
"instead."

#: src/vigilo/vigiconf/commandline.py:318
msgid "Unable to switch to user 'vigiconf'. Aborting."
msgstr "Unable to switch to user 'vigiconf'. Aborting."

#: src/vigilo/vigiconf/commandline.py:342
msgid "VigiConf was not launched as user 'vigiconf'. Aborting."
msgstr "VigiConf was not launched as user 'vigiconf'. Aborting."

#: src/vigilo/vigiconf/commandline.py:375
#, python-format
msgid ""
"Can't obtain lock on lockfile (%(lockfile)s). VigiConf already running ? "
"REASON : %(error)s"
msgstr ""
"Can't obtain lock on lockfile (%(lockfile)s). VigiConf already running ? "
"REASON : %(error)s"

#: src/vigilo/vigiconf/commandline.py:388
#: src/vigilo/vigiconf/commandline.py:390
#, python-format
msgid "VigiConf error: %s"
msgstr "VigiConf error: %s"

#: src/vigilo/vigiconf/commandline.py:394
msgid "VigiConf is done."
msgstr "VigiConf is done."

#: src/vigilo/vigiconf/conf.py:74
msgid "Loading XML configuration"
msgstr "Loading XML configuration"

#: src/vigilo/vigiconf/conf.py:79
msgid "Error loading configuration"
msgstr "Error loading configuration"

#: src/vigilo/vigiconf/discoverator.py:78
#, python-format
msgid "%s: No such file"
msgstr "%s: No such file"

#: src/vigilo/vigiconf/discoverator.py:90
#, python-format
msgid ""
"Invalid HOSTNAME detected in SNMPv2-MIB::sysName.0: \"%s\" (blank "
"character not allowed)"
msgstr ""
"Invalid HOSTNAME detected in SNMPv2-MIB::sysName.0: \"%s\" (blank "
"character not allowed)"

#: src/vigilo/vigiconf/discoverator.py:125
msgid "The \"snmpwalk\" command is not installed."
msgstr "The \"snmpwalk\" command is not installed."

#: src/vigilo/vigiconf/discoverator.py:128
#, python-format
msgid ""
"SNMP walk command failed with error status %(status)s and message:\n"
"%(msg)s\n"
"The command was: %(cmd)s"
msgstr ""
"SNMP walk command failed with error status %(status)s and message:\n"
"%(msg)s\n"
"The command was: %(cmd)s"

#: src/vigilo/vigiconf/applications/nagios_hls/generator.py:71
msgid "No server has been configured to host High-Level Services"
msgstr "No server has been configured to host High-Level Services"

#: src/vigilo/vigiconf/applications/vigirrd/generator.py:83
#, python-format
msgid "Some datasources are not on a graph: %s"
msgstr "Some datasources are not on a graph: %s"

#: src/vigilo/vigiconf/connector/command.py:28
msgid "An instance of Command can only be run once"
msgstr "An instance of Command can only be run once"

#: src/vigilo/vigiconf/connector/command.py:94
#, python-format
msgid "VigiConf finished properly (duration: %ds)"
msgstr "VigiConf finished properly (duration: %ds)"

#: src/vigilo/vigiconf/connector/command.py:99
#, python-format
msgid "VigiConf failed ! Exit code: %(rcode)d, duration: %(duration)ds"
msgstr "VigiConf failed ! Exit code: %(rcode)d, duration: %(duration)ds"

#: src/vigilo/vigiconf/connector/xmpptovigiconf.py:45
#, python-format
msgid "Recieved unhandled message type: %(msgtype)s"
msgstr "Recieved unhandled message type: %(msgtype)s"

#: src/vigilo/vigiconf/connector/xmpptovigiconf.py:75
#, python-format
msgid "Running command: %s"
msgstr "Running command: %s"

#: src/vigilo/vigiconf/connector/xmpptovigiconf.py:81
msgid "FAILED!"
msgstr "FAILED!"

#: src/vigilo/vigiconf/connector/xmpptovigiconf.py:84
msgid "OK."
msgstr "OK."

#: src/vigilo/vigiconf/connector/xmpptovigiconf.py:85
#, python-format
msgid "Duration: %d seconds"
msgstr "Duration: %d seconds"

#: src/vigilo/vigiconf/connector/xmpptovigiconf.py:155
#, python-format
msgid "Accepting subscription request from %s"
msgstr "Accepting subscription request from %s"

#: src/vigilo/vigiconf/lib/application.py:159
#, python-format
msgid "Can't find the validation script for app %(app)s: %(error)s"
msgstr "Can't find the validation script for app %(app)s: %(error)s"

#: src/vigilo/vigiconf/lib/application.py:275
#, python-format
msgid "%(app)s: validation failed for server '%(server)s': %(reason)s"
msgstr "%(app)s: validation failed for server '%(server)s': %(reason)s"

#: src/vigilo/vigiconf/lib/application.py:282
#, python-format
msgid "%(app)s : Validation successful for server: %(server)s"
msgstr "%(app)s : Validation successful for server: %(server)s"

#: src/vigilo/vigiconf/lib/application.py:303
#, python-format
msgid "%(app)s : Qualification failed on '%(server)s' - REASON: %(reason)s"
msgstr "%(app)s : Qualification failed on '%(server)s' - REASON: %(reason)s"

#: src/vigilo/vigiconf/lib/application.py:312
#, python-format
msgid "%(app)s : Qualification successful on server : %(server)s"
msgstr "%(app)s : Qualification successful on server : %(server)s"

#: src/vigilo/vigiconf/lib/application.py:325
#, python-format
msgid "Starting %(app)s on %(server)s ..."
msgstr "Starting %(app)s on %(server)s ..."

#: src/vigilo/vigiconf/lib/application.py:335
#, python-format
msgid "Can't Start %(app)s on %(server)s - REASON %(reason)s"
msgstr "Can't Start %(app)s on %(server)s - REASON %(reason)s"

#: src/vigilo/vigiconf/lib/application.py:343
#, python-format
msgid "%(app)s started on %(server)s"
msgstr "%(app)s started on %(server)s"

#: src/vigilo/vigiconf/lib/application.py:357
#, python-format
msgid "Stopping %(app)s on %(server)s ..."
msgstr "Stopping %(app)s on %(server)s ..."

#: src/vigilo/vigiconf/lib/application.py:367
#, python-format
msgid "Can't Stop %(app)s on %(server)s - REASON %(reason)s"
msgstr "Can't Stop %(app)s on %(server)s - REASON %(reason)s"

#: src/vigilo/vigiconf/lib/application.py:375
#, python-format
msgid "%(app)s stopped on %(server)s"
msgstr "%(app)s stopped on %(server)s"

#: src/vigilo/vigiconf/lib/application.py:407
#, python-format
msgid "%(app)s: %(action)s process failed."
msgstr "%(app)s: %(action)s process failed."

#: src/vigilo/vigiconf/lib/dispatchator.py:138
#, python-format
msgid ""
"Incoherent configuration: application %(app)s has an entry point named "
"%(epname)s in package %(eppkg)s"
msgstr ""
"Incoherent configuration: application %(app)s has an entry point named "
"%(epname)s in package %(eppkg)s"

#: src/vigilo/vigiconf/lib/dispatchator.py:143
#, python-format
msgid "Application %s is installed but disabled (see conf.d/general/apps.py)"
msgstr "Application %s is installed but disabled (see conf.d/general/apps.py)"

#: src/vigilo/vigiconf/lib/dispatchator.py:147
#, python-format
msgid "application %s is not unique."
msgstr "application %s is not unique."

#: src/vigilo/vigiconf/lib/dispatchator.py:148
#, python-format
msgid " Providing modules: %s"
msgstr " Providing modules: %s"

#: src/vigilo/vigiconf/lib/dispatchator.py:158
#, python-format
msgid ""
"Application %s has been added to conf.d/general/apps.py, but is not "
"installed"
msgstr ""
"Application %s has been added to conf.d/general/apps.py, but is not "
"installed"

#: src/vigilo/vigiconf/lib/dispatchator.py:183
msgid "Generation failed!"
msgstr "Generation failed!"

#: src/vigilo/vigiconf/lib/dispatchator.py:186
msgid "Generation successful"
msgstr "Generation successful"

#: src/vigilo/vigiconf/lib/dispatchator.py:193
msgid "SVN commit successful"
msgstr "SVN commit successful"

#: src/vigilo/vigiconf/lib/dispatchator.py:206
msgid "validation failed, see above for more information."
msgstr "validation failed, see above for more information."

#: src/vigilo/vigiconf/lib/dispatchator.py:208
msgid "Validation successful"
msgstr "Validation successful"

#: src/vigilo/vigiconf/lib/dispatchator.py:218
msgid ""
"you can't go back to a former revision if you have modified your "
"configuration. Use 'svn revert' to cancel your modifications"
msgstr ""
"you can't go back to a former revision if you have modified your "
"configuration. Use 'svn revert' to cancel your modifications"

#: src/vigilo/vigiconf/lib/dispatchator.py:234
#, python-format
msgid "can't get the SVN status for the configuration dir: %s"
msgstr "can't get the SVN status for the configuration dir: %s"

#: src/vigilo/vigiconf/lib/dispatchator.py:264
msgid "Not updating because the 'svnrepository' configuration parameter is empty"
msgstr "Not updating because the 'svnrepository' configuration parameter is empty"

#: src/vigilo/vigiconf/lib/dispatchator.py:287
#, python-format
msgid "Can't add %(path)s in repository: %(error)s"
msgstr "Can't add %(path)s in repository: %(error)s"

#: src/vigilo/vigiconf/lib/dispatchator.py:305
#, python-format
msgid "can't remove %(path)s from repository: %(error)s"
msgstr "can't remove %(path)s from repository: %(error)s"

#: src/vigilo/vigiconf/lib/dispatchator.py:313
msgid ""
"Not committing because the 'svnrepository' configuration parameter is "
"empty"
msgstr ""
"Not committing because the 'svnrepository' configuration parameter is "
"empty"

#: src/vigilo/vigiconf/lib/dispatchator.py:325
#, python-format
msgid "can't commit the configuration dir in SVN: %s"
msgstr "can't commit the configuration dir in SVN: %s"

#: src/vigilo/vigiconf/lib/dispatchator.py:340
#, python-format
msgid ""
"can't execute the request to update the local copy. COMMAND %(cmd)s "
"FAILED. REASON: %(reason)s"
msgstr ""
"can't execute the request to update the local copy. COMMAND %(cmd)s "
"FAILED. REASON: %(reason)s"

#: src/vigilo/vigiconf/lib/dispatchator.py:382
#, python-format
msgid "can't execute the request to get the current revision: %s"
msgstr "can't execute the request to get the current revision: %s"

#: src/vigilo/vigiconf/lib/dispatchator.py:425
msgid ""
"The configurations files have not been transfered on every server. See "
"above for more information."
msgstr ""
"The configurations files have not been transfered on every server. See "
"above for more information."

#: src/vigilo/vigiconf/lib/dispatchator.py:459
msgid "qualification failed. See above for more information."
msgstr "qualification failed. See above for more information."

#: src/vigilo/vigiconf/lib/dispatchator.py:461
msgid "Qualification successful"
msgstr "Qualification successful"

#: src/vigilo/vigiconf/lib/dispatchator.py:485
msgid "All servers are up-to-date, no deployment needed."
msgstr "All servers are up-to-date, no deployment needed."

#: src/vigilo/vigiconf/lib/dispatchator.py:494
msgid "Database commit successful"
msgstr "Database commit successful"

#: src/vigilo/vigiconf/lib/dispatchator.py:498
msgid "Database commit failed"
msgstr "Database commit failed"

#: src/vigilo/vigiconf/lib/dispatchator.py:546
msgid "Switch directories was not successful on each server."
msgstr "Switch directories was not successful on each server."

#: src/vigilo/vigiconf/lib/dispatchator.py:573
msgid "Stop applications failed"
msgstr "Stop applications failed"

#: src/vigilo/vigiconf/lib/dispatchator.py:586
msgid "Start applications failed"
msgstr "Start applications failed"

#: src/vigilo/vigiconf/lib/dispatchator.py:600
msgid "All servers are up-to-date. No restart needed."
msgstr "All servers are up-to-date. No restart needed."

#: src/vigilo/vigiconf/lib/dispatchator.py:610
#, python-format
msgid "Current revision in the repository : %d"
msgstr "Current revision in the repository : %d"

#: src/vigilo/vigiconf/lib/dispatchator.py:618
msgid "(should be deployed)"
msgstr "(should be deployed)"

#: src/vigilo/vigiconf/lib/dispatchator.py:620
msgid "(should restart)"
msgstr "(should restart)"

#: src/vigilo/vigiconf/lib/dispatchator.py:621
#, python-format
msgid "Revisions for server %(server)s : %(rev)s%(dep)s%(restart)s"
msgstr "Revisions for server %(server)s : %(rev)s%(dep)s%(restart)s"

#: src/vigilo/vigiconf/lib/dispatchator.py:627
#, python-format
msgid "Cannot get revision for server: %(server)s. REASON : %(reason)s"
msgstr "Cannot get revision for server: %(server)s. REASON : %(reason)s"

#: src/vigilo/vigiconf/lib/remotecommand.py:117
msgid "Server name set to None."
msgstr "Server name set to None."

#: src/vigilo/vigiconf/lib/remotecommand.py:119
msgid "Server name incorrect (length = 0)."
msgstr "Server name incorrect (length = 0)."

#: src/vigilo/vigiconf/lib/remotecommand.py:179
msgid "Unknown command type."
msgstr "Unknown command type."

#: src/vigilo/vigiconf/lib/remotecommand.py:196
#, python-format
msgid "The source file does not exist: %s"
msgstr "The source file does not exist: %s"

#: src/vigilo/vigiconf/lib/remotecommand.py:213
#: src/vigilo/vigiconf/lib/remotecommand.py:246
msgid "Cannot reach server."
msgstr "Cannot reach server."

#: src/vigilo/vigiconf/lib/remotecommand.py:216
#: src/vigilo/vigiconf/lib/remotecommand.py:248
msgid "Cannot find file."
msgstr "Cannot find file."

#: src/vigilo/vigiconf/lib/revisionmanager.py:87
#, python-format
msgid "Deployed: %d"
msgstr "Deployed: %d"

#: src/vigilo/vigiconf/lib/revisionmanager.py:88
#, python-format
msgid "Installed: %d"
msgstr "Installed: %d"

#: src/vigilo/vigiconf/lib/revisionmanager.py:89
#, python-format
msgid "Previous: %d"
msgstr "Previous: %d"

#: src/vigilo/vigiconf/lib/revisionmanager.py:176
#, python-format
msgid "Cannot write the revision file: %s"
msgstr "Cannot write the revision file: %s"

#: src/vigilo/vigiconf/lib/server.py:83
msgid "On the Community Edition, you can only use localhost"
msgstr "On the Community Edition, you can only use localhost"

#: src/vigilo/vigiconf/lib/server.py:178
#, python-format
msgid ""
"Can't activate the configuration on %(server)s. COMMAND %(cmd)s FAILED. "
"REASON: %(reason)s"
msgstr ""
"Can't activate the configuration on %(server)s. COMMAND %(cmd)s FAILED. "
"REASON: %(reason)s"

#: src/vigilo/vigiconf/lib/server.py:204
#, python-format
msgid "Can't tar config for server %(server)s: %(error)s"
msgstr "Can't tar config for server %(server)s: %(error)s"

#: src/vigilo/vigiconf/lib/server.py:222
#, python-format
msgid "%s : deployement successful."
msgstr "%s : deployement successful."

#: src/vigilo/vigiconf/lib/server.py:240
#, python-format
msgid "Cannot copy files (%(from)s to %(to)s): %(error)s."
msgstr "Cannot copy files (%(from)s to %(to)s): %(error)s."

#: src/vigilo/vigiconf/lib/systemcommand.py:52
#, python-format
msgid "Command \"%(cmd)s\" failed with code %(code)s and message: %(msg)s"
msgstr "Command \"%(cmd)s\" failed with code %(code)s and message: %(msg)s"

#: src/vigilo/vigiconf/lib/validator.py:125
msgid "No server configuration to be generated"
msgstr "No server configuration to be generated"

#: src/vigilo/vigiconf/lib/validator.py:128
msgid "No application configuration to be generated"
msgstr "No application configuration to be generated"

#: src/vigilo/vigiconf/lib/validator.py:131
msgid "No host configuration to be generated"
msgstr "No host configuration to be generated"

#: src/vigilo/vigiconf/lib/validator.py:139
#, python-format
msgid ""
"The number of host entries in the database does not match the number of "
"hosts in the configuration. Database: %(db)d, configuration: %(conf)d"
msgstr ""
"The number of host entries in the database does not match the number of "
"hosts in the configuration. Database: %(db)d, configuration: %(conf)d"

#: src/vigilo/vigiconf/lib/validator.py:167
#, python-format
msgid ""
"The number of services entries in the database does not match the number "
"of services in the configuration. Database: %(db)d, configuration: "
"%(conf)d"
msgstr ""
"The number of services entries in the database does not match the number "
"of services in the configuration. Database: %(db)d, configuration: "
"%(conf)d"

#: src/vigilo/vigiconf/lib/validator.py:181
#, python-format
msgid ""
"The number of apps loaded in the database does not match the number of "
"apps to deploy. Database: %(db)d, to deploy: %(deploy)d. Possible cause: "
"one or more app have no Vigilo server to be deployed on."
msgstr ""
"The number of apps loaded in the database does not match the number of "
"apps to deploy. Database: %(db)d, to deploy: %(deploy)d. Possible cause: "
"one or more app have no Vigilo server to be deployed on."

#: src/vigilo/vigiconf/lib/validator.py:199
#, python-format
msgid ""
"The number of ventilation entries in the database does not match the "
"number of host and apps. Database: %(db)d, should be: %(conf)d. Possible "
"cause: one or more application have no Vigilo server to be deployed on."
msgstr ""
"The number of ventilation entries in the database does not match the "
"number of host and apps. Database: %(db)d, should be: %(conf)d. Possible "
"cause: one or more application have no Vigilo server to be deployed on."

#: src/vigilo/vigiconf/lib/validator.py:233
#, python-format
msgid ""
"%(nb_gens)d generators exist whereas %(nb_apps)d applications have been "
"configured"
msgstr ""
"%(nb_gens)d generators exist whereas %(nb_apps)d applications have been "
"configured"

#: src/vigilo/vigiconf/lib/validator.py:240
#, python-format
msgid "########> Error in %(source)s (object %(object)s): %(error)s"
msgstr "########> Error in %(source)s (object %(object)s): %(error)s"

#: src/vigilo/vigiconf/lib/validator.py:247
#, python-format
msgid "====> Warning in %(source)s (object %(object)s): %(error)s"
msgstr "====> Warning in %(source)s (object %(object)s): %(error)s"

#: src/vigilo/vigiconf/lib/validator.py:253
#, python-format
msgid "%d errors"
msgstr "%d errors"

#: src/vigilo/vigiconf/lib/validator.py:254
#, python-format
msgid "%d warnings"
msgstr "%d warnings"

#: src/vigilo/vigiconf/lib/validator.py:256
#, python-format
msgid ""
"%(nbGenerators)d generators have configured %(nbHosts)d hosts on "
"%(nbServers)d servers."
msgstr ""
"%(nbGenerators)d generators have configured %(nbHosts)d hosts on "
"%(nbServers)d servers."

#: src/vigilo/vigiconf/lib/validator.py:258
#, python-format
msgid "%(filesWritten)d files have been generated in %(dirsCreated)d directories"
msgstr "%(filesWritten)d files have been generated in %(dirsCreated)d directories"

#: src/vigilo/vigiconf/lib/confclasses/host.py:136
#: src/vigilo/vigiconf/lib/confclasses/hosttemplate.py:159
#, python-format
msgid "The \"%s\" attribute has been deprecated. Please use \"%s\" instead."
msgstr "The \"%s\" attribute has been deprecated. Please use \"%s\" instead."

#: src/vigilo/vigiconf/lib/confclasses/host.py:185
#, python-format
msgid ""
"Test \"%(test_name)s\" on \"%(host)s\" needs the following arguments: "
"%(args)s (and only those)"
msgstr ""
"Test \"%(test_name)s\" on \"%(host)s\" needs the following arguments: "
"%(args)s (and only those)"

#: src/vigilo/vigiconf/lib/confclasses/host.py:541
#, python-format
msgid "Wrong datasource in graph '%(graph)s': %(ds)s"
msgstr "Wrong datasource in graph '%(graph)s': %(ds)s"

#: src/vigilo/vigiconf/lib/confclasses/host.py:773
#: src/vigilo/vigiconf/lib/confclasses/hosttemplate.py:278
#: src/vigilo/vigiconf/lib/loaders/xmlloader.py:93
#, python-format
msgid "Invalid XML validation schema %(schema)s: %(error)s"
msgstr "Invalid XML validation schema %(schema)s: %(error)s"

#: src/vigilo/vigiconf/lib/confclasses/host.py:779
#: src/vigilo/vigiconf/lib/confclasses/host.py:799
#: src/vigilo/vigiconf/lib/confclasses/hosttemplate.py:284
#: src/vigilo/vigiconf/lib/confclasses/hosttemplate.py:304
#: src/vigilo/vigiconf/lib/loaders/xmlloader.py:99
#: src/vigilo/vigiconf/lib/loaders/xmlloader.py:119
#, python-format
msgid ""
"Error reading %(file)s, make sure the permissions are set "
"correctly.Message: %(error)s."
msgstr ""
"Error reading %(file)s, make sure the permissions are set "
"correctly.Message: %(error)s."

#: src/vigilo/vigiconf/lib/confclasses/host.py:796
#: src/vigilo/vigiconf/lib/confclasses/hosttemplate.py:301
#: src/vigilo/vigiconf/lib/loaders/xmlloader.py:116
#, python-format
msgid "XML syntax error in %(file)s: %(error)s"
msgstr "XML syntax error in %(file)s: %(error)s"

#: src/vigilo/vigiconf/lib/confclasses/host.py:807
#: src/vigilo/vigiconf/lib/confclasses/hosttemplate.py:312
#: src/vigilo/vigiconf/lib/loaders/xmlloader.py:127
#, python-format
msgid "XML validation failed: %(error)s"
msgstr "XML validation failed: %(error)s"

#: src/vigilo/vigiconf/lib/confclasses/host.py:859
#, python-format
msgid "Invalid ventilation group: %s"
msgstr "Invalid ventilation group: %s"

#: src/vigilo/vigiconf/lib/confclasses/host.py:893
#, python-format
msgid "Invalid weight value for test %(test)s on host %(host)s: %(weight)r"
msgstr "Invalid weight value for test %(test)s on host %(host)s: %(weight)r"

#: src/vigilo/vigiconf/lib/confclasses/host.py:941
#: src/vigilo/vigiconf/lib/confclasses/hosttemplate.py:119
#: src/vigilo/vigiconf/loaders/hlservice.py:143
#, python-format
msgid "Invalid group name (%s)"
msgstr "Invalid group name (%s)"

#: src/vigilo/vigiconf/lib/confclasses/host.py:950
#, python-format
msgid "Invalid weight value for host %(host)s: %(weight)r"
msgstr "Invalid weight value for host %(host)s: %(weight)r"

#: src/vigilo/vigiconf/lib/confclasses/host.py:963
#, python-format
msgid "You must associate host \"%s\" with at least one group."
msgstr "You must associate host \"%s\" with at least one group."

#: src/vigilo/vigiconf/lib/confclasses/hosttemplate.py:389
#, python-format
msgid "Invalid weight value for test %(test)s in template %(tpl)s: %(weight)r"
msgstr "Invalid weight value for test %(test)s in template %(tpl)s: %(weight)r"

#: src/vigilo/vigiconf/lib/confclasses/hosttemplate.py:411
#, python-format
msgid "Invalid weight value for template %(tpl)s: %(weight)r"
msgstr "Invalid weight value for template %(tpl)s: %(weight)r"

#: src/vigilo/vigiconf/lib/dispatchmodes/__init__.py:54
msgid ""
"You are trying remote deployment on the Community edition. This feature "
"is only available in the Enterprise edition. Aborting."
msgstr ""
"You are trying remote deployment on the Community edition. This feature "
"is only available in the Enterprise edition. Aborting."

#: src/vigilo/vigiconf/lib/dispatchmodes/remote.py:91
#, python-format
msgid "The %s app group is not listed in appsGroupsByServer"
msgstr "The %s app group is not listed in appsGroupsByServer"

#: src/vigilo/vigiconf/lib/dispatchmodes/remote.py:111
#, python-format
msgid "Invalid server name: %(servername)s. Available servers: %(servers)s"
msgstr "Invalid server name: %(servername)s. Available servers: %(servers)s"

#: src/vigilo/vigiconf/lib/generators/base.py:79
msgid "Generators must define the generate_host() method"
msgstr "Generators must define the generate_host() method"

#: src/vigilo/vigiconf/lib/generators/file.py:96
#, python-format
msgid "No such template: %s"
msgstr "No such template: %s"

#: src/vigilo/vigiconf/lib/generators/manager.py:100
#: src/vigilo/vigiconf/lib/generators/manager.py:310
#, python-format
msgid "Generated configuration for %s"
msgstr "Generated configuration for %s"

#: src/vigilo/vigiconf/lib/generators/manager.py:129
msgid "Computing ventilation"
msgstr "Computing ventilation"

#: src/vigilo/vigiconf/lib/generators/manager.py:144
msgid "Running generators"
msgstr "Running generators"

#: src/vigilo/vigiconf/lib/generators/manager.py:166
#, python-format
msgid "Generating configuration for %s"
msgstr "Generating configuration for %s"

#: src/vigilo/vigiconf/lib/generators/manager.py:172
msgid "Running database generators"
msgstr "Running database generators"

#: src/vigilo/vigiconf/lib/generators/manager.py:185
#, python-format
msgid "%(errtype)s in application %(app)s: %(error)s"
msgstr "%(errtype)s in application %(app)s: %(error)s"

#: src/vigilo/vigiconf/lib/generators/manager.py:206
msgid "Can't find the metro server for the RRD-based services"
msgstr "Can't find the metro server for the RRD-based services"

#: src/vigilo/vigiconf/lib/generators/manager.py:218
#: src/vigilo/vigiconf/lib/generators/manager.py:228
#, python-format
msgid ""
"The metrology server %s must be supervised for the RRD-based services to "
"work"
msgstr ""
"The metrology server %s must be supervised for the RRD-based services to "
"work"

#: src/vigilo/vigiconf/lib/generators/map.py:78
#, python-format
msgid "The map group %s has not been added by the install procedure"
msgstr "The map group %s has not been added by the install procedure"

#: src/vigilo/vigiconf/lib/loaders/dbloader.py:61
msgid "The 'load' method must be redefined"
msgstr "The 'load' method must be redefined"

#: src/vigilo/vigiconf/lib/loaders/dbloader.py:115
msgid ""
"The key attribute must be defined, or the get_key() method must be "
"redefined"
msgstr ""
"The key attribute must be defined, or the get_key() method must be "
"redefined"

#: src/vigilo/vigiconf/lib/loaders/xmlloader.py:88
#, python-format
msgid "XSD file does not exist: %s"
msgstr "XSD file does not exist: %s"

#: src/vigilo/vigiconf/lib/loaders/xmlloader.py:178
#, python-format
msgid "End tag mismatch error: %(start)s/%(end)s"
msgstr "End tag mismatch error: %(start)s/%(end)s"

#: src/vigilo/vigiconf/lib/loaders/xmlloader.py:185
#, python-format
msgid "Syntax error in \"%(path)s\": %(error)s"
msgstr "Syntax error in \"%(path)s\": %(error)s"

#: src/vigilo/vigiconf/lib/loaders/xmlloader.py:354
#, python-format
msgid "Can't find an item matching %s"
msgstr "Can't find an item matching %s"

#: src/vigilo/vigiconf/lib/servertypes/local.py:46
#, python-format
msgid "Can't deploy the tar archive for server %(server)s: %(error)s"
msgstr "Can't deploy the tar archive for server %(server)s: %(error)s"

#: src/vigilo/vigiconf/lib/servertypes/remote.py:50
#, python-format
msgid "Cannot find SSH config file: %s"
msgstr "Cannot find SSH config file: %s"

#: src/vigilo/vigiconf/lib/servertypes/remote.py:87
#, python-format
msgid "The archive file does not exist: %s"
msgstr "The archive file does not exist: %s"

#: src/vigilo/vigiconf/lib/servertypes/remote.py:93
#, python-format
msgid "Can't copy the configuration archive to %(server)s: %(error)s"
msgstr "Can't copy the configuration archive to %(server)s: %(error)s"

#: src/vigilo/vigiconf/lib/servertypes/remote.py:108
#, python-format
msgid "Can't deploy the configuration for server %(server)s: %(error)s"
msgstr "Can't deploy the configuration for server %(server)s: %(error)s"

#: src/vigilo/vigiconf/lib/ventilation/remote.py:147
#, python-format
msgid ""
"Could not determine how to ventilate host \"%s\". Affect some groups to "
"this host or use the ventilation attribute."
msgstr ""
"Could not determine how to ventilate host \"%s\". Affect some groups to "
"this host or use the ventilation attribute."

#: src/vigilo/vigiconf/lib/ventilation/remote.py:153
#, python-format
msgid ""
"Found multiple candidates for ventilation (%(candidates)r) on "
"\"%(host)s\", use the ventilation attribute to select one."
msgstr ""
"Found multiple candidates for ventilation (%(candidates)r) on "
"\"%(host)s\", use the ventilation attribute to select one."

#: src/vigilo/vigiconf/lib/ventilation/remote.py:258
#, python-format
msgid ""
"No server available for the appgroup %(appgroup)s and the hostgroup "
"%(hostgroup)s, skipping it"
msgstr ""
"No server available for the appgroup %(appgroup)s and the hostgroup "
"%(hostgroup)s, skipping it"

#: src/vigilo/vigiconf/lib/ventilation/remote.py:271
#, python-format
msgid ""
"Trying to ventilate hostgroup %(hostgroup)s for appgroup %(appgroup)s, "
"but the mapping is not in appsGroupsByServer."
msgstr ""
"Trying to ventilate hostgroup %(hostgroup)s for appgroup %(appgroup)s, "
"but the mapping is not in appsGroupsByServer."

#: src/vigilo/vigiconf/lib/ventilation/remote.py:326
#: src/vigilo/vigiconf/lib/ventilation/remote.py:342
#, python-format
msgid "The Vigilo server %s does not exist"
msgstr "The Vigilo server %s does not exist"

#: src/vigilo/vigiconf/lib/ventilation/remote.py:329
#, python-format
msgid "The Vigilo server %s is already disabled"
msgstr "The Vigilo server %s is already disabled"

#: src/vigilo/vigiconf/lib/ventilation/remote.py:345
#, python-format
msgid "The Vigilo server %s is already enabled"
msgstr "The Vigilo server %s is already enabled"

#: src/vigilo/vigiconf/loaders/group.py:134
msgid "Loading groups"
msgstr "Loading groups"

#: src/vigilo/vigiconf/loaders/hlservice.py:99
#, python-format
msgid "Unknown group \"%(group)s\" in HLS \"%(hls)s\"."
msgstr "Unknown group \"%(group)s\" in HLS \"%(hls)s\"."

#: src/vigilo/vigiconf/loaders/hlservice.py:133
#, python-format
msgid "Invalid operator \"%(operator)s\". Possible values: %(possibilities)s"
msgstr "Invalid operator \"%(operator)s\". Possible values: %(possibilities)s"

#: src/vigilo/vigiconf/loaders/hlservice.py:183
#, python-format
msgid ""
"syntax error in HLS %(hls)s: could not find a group matching path "
"\"%(path)s\""
msgstr ""
"syntax error in HLS %(hls)s: could not find a group matching path "
"\"%(path)s\""

#: src/vigilo/vigiconf/loaders/host.py:71
#, python-format
msgid "Deleting leftover config file from database: %s"
msgstr "Deleting leftover config file from database: %s"

#: src/vigilo/vigiconf/loaders/host.py:77
msgid "Loading hosts"
msgstr "Loading hosts"

#: src/vigilo/vigiconf/loaders/host.py:128
#, python-format
msgid "Loading host %s"
msgstr "Loading host %s"

#: src/vigilo/vigiconf/loaders/host.py:241
msgid "Done loading hosts"
msgstr "Done loading hosts"

#: src/vigilo/vigiconf/loaders/host.py:264
#, python-format
msgid "Unknown group \"%(group)s\" in host \"%(host)s\"."
msgstr "Unknown group \"%(group)s\" in host \"%(host)s\"."

#: src/vigilo/vigiconf/loaders/host.py:291
#, python-format
msgid ""
"syntax error in host %(host)s: could not find a group matching path "
"\"%(path)s\""
msgstr ""
"syntax error in host %(host)s: could not find a group matching path "
"\"%(path)s\""

#: src/vigilo/vigiconf/loaders/topology.py:102
#, python-format
msgid "Not a Dependency: %s"
msgstr "Not a Dependency: %s"

#: src/vigilo/vigiconf/loaders/topology.py:117
msgid "No dependent supervised item"
msgstr "No dependent supervised item"

#: src/vigilo/vigiconf/loaders/ventilation.py:143
#, python-format
msgid ""
"The application %s has been removed from the configuration, it needs to "
"be stopped manually"
msgstr ""
"The application %s has been removed from the configuration, it needs to "
"be stopped manually"

